src/components/functional_module/include/functional_module/plugin_manager.h

```diff
@@ -48,15 +48,27 @@ class PluginManager : public ModuleObserver {
   typedef std::map<ModuleID, ModulePtr> Modules;
   int LoadPlugins(const std::string& plugin_path);
   void UnloadPlugins();
-  void ProcessMessage(application_manager::MessagePtr msg);
+
+  /**
+   * @brief ProcessMessage forwards modules message to modules if any is
```

Comment:
@dev-gh probably "ProcessMessage forwards MOBILE message to modules"

Reply:
@AKalinich-Luxoft 
Done with f31f64f